pkg/handler/v2/routes_v2.go

```diff
@@ -4676,6 +4681,36 @@ func (r Routing) listUser() http.Handler {
 	)
 }
 
+// swagger:route GET /api/v2/featuregates get status of feature gates
+//
+//     Status of feature gates
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       default: errorResponse
+//       200: FeatureGate
+//       401: errorResponse
+//       403: errorResponse
+func (r Routing) getFeatureGates() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(
+			func(ctx context.Context, request interface{}) (interface{}, error) {
```

Comment:
please add this method to `kubermatic/pkg/handler/v2` as a new folder with file `feature_gates/feature_gates.go`